src/components/page-active-job-component/style.css

```diff
@@ -1,6 +1,6 @@
-.scheduled-job {
-  display: grid;
-  grid-auto-flow: column;
-  grid-auto-columns: max-content;
-  gap: var(--eds_spacing_medium);
+.scheduled-job__link {
+  text-decoration: none;
```

Comment:
When hovering the scheduled job link, the underline is blue. So this class should probably use the interactive_primary color. 
![image](https://user-images.githubusercontent.com/79976935/131461676-e7cb7c09-679b-46b5-a959-35bbe0d01335.png)

Reply:
fixed